integration/spark/src/main/scala/org/apache/spark/util/AlterTableUtil.scala

```diff
@@ -916,6 +930,19 @@ object AlterTableUtil {
         }
       }
     }
+    // should not be present in index tables
+    val indexTables = CarbonIndexUtil.getSecondaryIndexes(carbonTable)
```

Comment:
Instead of getting from index table, can directly get the index Column names from carbonTable.getIndexesMap.get(IndexType.SI.getIndexProviderName) -> CarbonCommonConstants.INDEX_COLUMNS

Reply:
done